build-toolchain.bash

```diff
@@ -51,6 +52,9 @@ function usage()
 
 for ARG in $*; do
 	case $ARG in
+		--prefix=*)
+			PREFIX="${ARG#*=}"
```

Comment:
Something like `PREFIX="${ARG#*=}/retro68"` would do the trick. Do you have a directory name that you would prefer?